lisa/wlgen/rta.py

```diff
@@ -726,10 +727,15 @@ class Phase(Loggable):
                           when reaching the end of this phase. Currently only
                           supported when duty_cycle_pct=100
     :type barrier_after: str
+
+    :param nodes_membind: the list of NUMA Nodes.
+        Task will only allocate memory from these nodes during this phase.
+        If unspecified, that phase will inherit memory allocation policy from previous phase.
```

Comment:
To follow the spirit of what @douglas-raillard-arm did in ed748695503e ("lisa.wlgen.rta: Align cpus phase parameter with its documentation"), we should probably reset the restriction if unspecified in the rt-app description of a given phase.

The main argument back then was that this prevents composition from being deterministic: if you add two rt-app tasks together you don't want the first phase of the second task to be impacted by the last phase of the first task.

Reply:
I will follow the same concept.